build-system/pr-check/nomodule-tests.js

```diff
@@ -32,12 +32,12 @@ const {runCiJob} = require('./ci-job');
 const jobName = 'nomodule-tests.js';
 
 function prependConfig() {
-  // TODO(@ampproject/wg-infra): change prepend-global to take multiple target files instead of looping here.
-  for (const target of MINIFIED_TARGETS) {
-    timedExecOrDie(
-      `gulp prepend-global --${process.env.config} --local_dev --fortesting --derandomize --target=dist/${target}.js`
-    );
-  }
+  const targets = MINIFIED_TARGETS.flatMap((target) => [
+    `dist/${target}.js`,
+  ]).join(',');
+  timedExecOrDie(
+    `gulp prepend-global --${process.env.config} --local_dev --fortesting --derandomize --target=${targets}`
+  );
```

Comment:
Since this function is similar to the one in `module-tests.js`, can we consolidate them into a single function that takes a `config` arg and make it live in `pr-check/utils.js`?

Reply:
How would we handle the `.js` vs `.js, .mjs` cases in such a function without making it too generic to be useful?

Reply:
I think we can create a single version for both cases. Something like this:

```js
function prependConfig(config) {
  const targets = MINIFIED_TARGETS.flatMap((target) => [
    `dist/${target}.js`,
    `dist/${target}.mjs`,
  ])
    .filter(fs.existsSync) // If this shorthand doesn't work, expand to ((file) => fs.existsSync(file))
    .join(',');
  timedExecOrDie(
    `gulp prepend-global --${config} --local_dev --fortesting --derandomize --target=${targets}`
  );
}
```

Reply:
I prefer not to run existsSync for this, it feels like clunky to me

Reply:
Missed this comment in a flood of notifications yesterday. I'm still of the opinion that this function should be de-duped and should live in a utility file instead of this task wrapper. Okay to leave this for a follow up PR though.